main.go

```diff
@@ -361,22 +377,43 @@ func main() {
 				failf("Failed to create temp dir, error: %s", err)
 			}
 
+			var mergedTestResultXmlPths []string
 			for fileName, fileURL := range responseModel {
-				if err := downloadFile(fileURL, filepath.Join(tempDir, fileName)); err != nil {
+				pth := filepath.Join(tempDir, fileName)
+				if err := downloadFile(fileURL, pth); err != nil {
 					failf("Failed to download file, error: %s", err)
 				}
+
+				// per test run results: iphone13pro-16.6-en-portrait_test_result_0.xml
+				// rerun test results: iphone8-16.6-en-portrait-rerun_1_test_result_0.xml
+				// merged result: iphone13pro-16.6-en-portrait-test_results_merged.xml
```

Comment:
Are these the filenames Firebase use for their test results? Do they mention the naming convention anywhere in their docs? Just wondering if this is an implementation detail and could change in the future on their side.